### PR TITLE
Added Crio in the list

### PR DESCRIPTION
**Pull request by sudhanshutiwari264**
Added Crio in the list as it is for Beginners !

Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by crweiner** [comment]
Hi @sudhanshutiwari264! :wave: This seems like a great addition to the Swag List! Can you please also add them in the "Least Invovement to Most Involvement" section of the list and take care of the Code Climate flags? 

Thanks! :smile:

**Comment by sudhanshutiwari264** [comment]
@crweiner Sure I will add it to the least involvement required and will also take the care of code flag !
And I willalso make little changes to it okay ?
Thanks!

**Comment by crweiner** [comment]
> @crweiner Sure I will add it to the least involvement required and will also take the care of code flag !
> And I willalso make little changes to it okay ?
> Thanks!

You go right ahead and make any changes you want! 😄 Let me know when you're ready for me to take a look. If you [follow the Contributing guide](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) then hopefully that helps you with some changes. 👍

**Comment by sudhanshutiwari264** [comment]
@crweiner I have updated it But can you please have a look if it is right!
I m also a beginner so please help me to resolve any type of issue I have tried my best to resolve them!

**Comment by sudhanshutiwari264** [comment]
@crweiner I have cleared all the issues and also added my name to contribition.md and finally all the issues are cleared!

File: README.md [hunk 1]
```diff
@@ -745,6 +752,13 @@ PRs must link to a related issue page (you can create the issue!) and follow Dru
 - **How to sign up**: Make PRs on [ArchivesSpace Github](https://github.com/archivesspace/archivesspace) and the top contributor will win the bag.
 - **Notes**: Check out [ArchivesSpace Hacktoberfest 2020 blog](https://archivesspace.org/archives/6563) for more information and help.
 
+#### **Crio**
+
+- **Swag**: Crio bag of goodies and T -shirt and upto 50% scholarship .
+- **Requirements**: Top 50 performers
+- **How to sign up**: Fill out the form [here](https://crio.do/IBelieveInDoing/October/?utm_source=adv&name=Sudhanshu) for participation ! more details will be given by emails !
+- **Notes**: For more details visit our [contributing guidelines](https://crio.do/IBelieveInDoing/October/?utm_source=adv&name=Sudhanshu) and dedicated [hacktoberfest page](https://crio.do/IBelieveInDoing/October/?utm_source=adv&name=Sudhanshu) to get involved.Only eligible contributors residing in India will be shipped stickers (including a limited edition Crio Hacktoberfest ones) and those making top 50 contributors will be shipped an exclusive t-shirt (within India only).
```
[the remark this answers is not in the document]
Comment (sudhanshutiwari264) [reply README.md:760]: LGTM means @LautaroLobo12?

[the remark this answers is not in the document]
Comment (LautaroLobo12) [reply README.md:760]: Looks Good To Me =)

[the remark this answers is not in the document]
Comment (sudhanshutiwari264) [reply README.md:760]: > Looks Good To Me =)

Thanks !

**Comment by crweiner** [comment]
Hey @sudhanshutiwari264! Thank you very much for your hard work in getting Crio on the 2020 Hacktoberfest Swag List! I appreciate you tackling the Code Climate flags and cleaning things up really quickly. :+1: Also, thank you @LautaroLobo12 for the extra pair of eyes and reviewing things for me! :smile:

I took a look at the [Crio Hacktoberfest page](https://www.crio.do/IBelieveInDoing/October) and I don't see a mention of them offering a t-shirt anymore and I also don't see that it is India-only. I'm assuming that India-only is correct and that maybe the t-shirt has just been replaced by "goodies"? If you know of any updates from them, please share!

**Comment by sudhanshutiwari264** [comment]
@crweiner thanks for reviewing again the goodies and t-shirt both will be given to the top contributors maybe they don't written it on there page but that's the case !

**Comment by crweiner** [comment]
Alrighty, sounds good to me! If they change anything, please feel free to submit another PR to update things. :smile: I appreciate your contribution and have a marvelous Hacktoberfest! :tada: :tada: :tada:

you rock

**Comment by sudhanshutiwari264** [comment]
@crweiner Sure bro I will update asap I can if anything updates okay !

**Comment by sudhanshutiwari264** [comment]
You tooo have a marvelous fantastic Hactoberfest !

**Comment by sudhanshutiwari264** [comment]
> Alrighty, sounds good to me! If they change anything, please feel free to submit another PR to update things. 😄 I appreciate your contribution and have a marvelous Hacktoberfest! 🎉 🎉 🎉
> 
> you rock

Bro can u change the name to Crio.do

**Pull request merged**